test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusIntegrationTestExtension.java

```diff
@@ -1,15 +1,6 @@
 package io.quarkus.test.junit;
 
 import static io.quarkus.test.junit.IntegrationTestUtil.*;
```

Comment:
Star imports are discouraged AFAIR

Reply:
OK, I can drop that star line and introduce back explicit definitions + 2 new static imports

Reply:
Done @gastaldi